easybuild/easyblocks/n/numpy.py

```diff
@@ -142,7 +142,7 @@ def get_libs_for_mkl(varname):
                 # patches are either strings (extension) or dicts (easyblock)
                 if isinstance(patch, dict):
                     patch = patch['path']
-                if patch_wl_regex.search(open(patch, 'r').read()):
+                if patch_wl_regex.search(read_file(patch)):
```

Comment:
hm, it's probably me but I don't see how this fixes it? doesn't the `read_file` function do essentially the same?

Reply:
@dithwick can you create a test report that includes this PR?

Reply:
Yep will do!